.gitignore

```diff
@@ -169,3 +169,5 @@ gfx/trainers/catman\.2bpp
 gfx/trainers/jessie_james\.2bpp
 
 install\(3\)\.bat
+
+tools/*
```

Comment:
This isn't right; some files in tools/ are not ignored. Did you want to ignore `*.pyc` or something?

Reply:
Running `make` generates stuff in the tool folder, and I don't think generated things should be pushed.